resources/views/community/components/user/progression-status/console-progression-list-item.blade.php

```diff
@@ -0,0 +1,98 @@
+@props([
+    'label' => null,
+    'consoleId' => 0,
+    'unfinishedCount' => 0,
+    'beatenSoftcoreCount' => 0,
+    'beatenHardcoreCount' => 0,
+    'completedCount' => 0,
+    'masteredCount' => 0,
+])
+
+<?php
+$gameSystemIconSrc = getSystemIconUrl($consoleId);
+
+$totalBeatenGamesCount = $beatenSoftcoreCount + $beatenHardcoreCount;
+$totalMasteredGamesCount = $completedCount + $masteredCount;
+$totalGamesCount = $unfinishedCount + $beatenSoftcoreCount + $beatenHardcoreCount + $completedCount + $masteredCount;
+
+$unfinishedGamesWidth = sprintf("%1.2f", ($unfinishedCount / $totalGamesCount) * 100.0);
+$beatenGamesWidth = sprintf("%1.2f", ($totalBeatenGamesCount / $totalGamesCount) * 100.0);
+$masteredGamesWidth = sprintf("%1.2f", ($totalMasteredGamesCount / $totalGamesCount) * 100.0);
+
+$widthsPreference = request()->cookie('progression_status_widths_preference');
+
+$widthMode = $widthsPreference;
+if ($widthMode !== 'equal' && $widthMode !== 'dynamic') {
+    $widthMode = 'equal';
+}
+
+$displayLabel = $label ?? config('systems')[$consoleId]['name_short'];
+$consoleTooltipLabel = $label ?? config('systems')[$consoleId]['name'];
+?>
+
+<li class="progression-status-row">
+    <a
+        href="#"
+        class="border-embed-highlight w-[102px] !min-w-[92px] pl-2 rounded-l hover:border-link-hover"
+        @if (!$label) title="{{ $consoleTooltipLabel }}" @endif
+    >
```

Comment:
From the PR description:
>NOTE: While the cells are indeed hyperlinks, they currently go nowhere. That will be handled in a subsequent PR.

Happy to elaborate more here. Building out the destination for these hyperlinks is my current task. Ideally the user is taken to a list of drilled-down results for each cell.

If you would prefer, we can disable these hyperlinks in this PR until this is ready, but I figured because this would be behind the feature flag anyway it may just be additional unneeded lift. Hopefully that makes sense.

Reply:
Apparently the bold made it harder for me to see. I read through everything again and still missed it. I had to Ctrl+F to find it :P

Reply:
No worries! I'll try to come up with some way to make these "caveats" stand out more in my PR descriptions when they come up.